src/CompareItem.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import './styles/App.css';
+
+const CompareItem = (props) => { 
```

Comment:
Tied to the compare.js file (render section), I am clear on the code below and don't have any changes.